dist/webapis-notification.min.js

```diff
@@ -1 +1,15 @@
-!function(o,t){"object"==typeof exports&&"undefined"!=typeof module?t():"function"==typeof define&&define.amd?define(t):t()}(0,function(){"use strict";Zone.__load_patch("notification",function(o,t,e){var n=o.Notification;if(n&&n.prototype){var i=Object.getOwnPropertyDescriptor(n.prototype,"onerror");i&&i.configurable&&e.patchOnProperties(n.prototype,null)}})});
\ No newline at end of file
+/**
+* @license
+* Copyright Google Inc. All Rights Reserved.
+*
+* Use of this source code is governed by an MIT-style license that can be
+* found in the LICENSE file at https://angular.io/license
+*/
+!function(t){"function"==typeof define&&define.amd?define(t):t()}(function(){"use strict";
+/**
```

Comment:
seems like we should figure out how to avoid having the license duplicated? I suppose it compresses well...

Reply:
The current `zone.js bundles` have the same issues, because inside the source code, we also have the `license header`, so I think we can resolve this issue later.